src/pages/workspace/accounting/xero/export/XeroExportConfigurationPage.tsx

```diff
@@ -0,0 +1,111 @@
+            error: errorFields?.exporter ? translate('common.genericErrorMessage') : undefined,
+        },
+        {
+            description: translate('workspace.xero.exportExpenses'),
+            title: translate('workspace.xero.purchaseBill'),
+            interactive: false,
+            shouldShowRightIcon: false,
+            helperText: translate('workspace.xero.exportExpensesDescription'),
+        },
+        {
+            description: translate('workspace.xero.purchaseBillDate'),
+            onPress: () => {},
+            brickRoadIndicator: errorFields?.billDate ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: exportConfiguration?.billDate,
+            pendingAction: pendingFields?.export,
+            error: errorFields?.billDate ? translate('common.genericErrorMessage') : undefined,
+        },
+        {
+            description: translate('workspace.xero.exportInvoices'),
+            title: translate('workspace.xero.salesInvoice'),
+            interactive: false,
+            shouldShowRightIcon: false,
+            helperText: translate('workspace.xero.exportInvoicesDescription'),
+        },
+        {
+            description: translate('workspace.xero.exportCompanyCard'),
+            title: translate('workspace.xero.bankTransactions'),
+            shouldShowRightIcon: false,
+            interactive: false,
+            helperText: translate('workspace.xero.exportDeepDiveCompanyCard'),
+        },
+        {
+            description: translate('workspace.xero.xeroBankAccount'),
+            onPress: () => {},
+            brickRoadIndicator: errorFields?.nonReimbursableAccount ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: undefined,
+            pendingAction: pendingFields?.export,
+            error: undefined,
+        },
+    ];
+
+    return (
+        <AccessOrNotFoundWrapper
+            policyID={policyID}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN]}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+        >
+            <ScreenWrapper
+                includeSafeAreaPaddingBottom={false}
+                testID={XeroExportConfigurationPage.displayName}
+            >
+                <HeaderWithBackButton title={translate('workspace.xero.export')} />
+                <ScrollView contentContainerStyle={styles.pb2}>
+                    <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.xero.exportDescription')}</Text>
+                    {menuItems.map((menuItem) => (
+                        <OfflineWithFeedback
+                            key={menuItem.description}
+                            pendingAction={menuItem.pendingAction}
+                        >
+                            <MenuItemWithTopDescription
+                                title={menuItem.title}
+                                interactive={menuItem?.interactive ?? true}
```

Comment:
Do we really need to define this for each menuItem? Can we not just set this false and change later if required ?

Reply:
yes because some items are interactive and some aren't

Reply:
I must’ve misread. Thanks for clarifying.